ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-access-lists.j2

```diff
@@ -0,0 +1,10 @@
+{% if ip_access_lists is arista.avd.defined %}
+
+## Extended Access-lists Device Configuration (Improved Data Model)
```

Comment:
This is the lazy documentation :) I understand that the value of a table will be limited.
Please change the heading to `### IP Access-lists Summary` and then add another heading above the configuration with `### IP Access-lists Configuration` and a `## IP Access-lists` at the top. Don't include the `improved data model` in the device documentation. We have that covered in the readme.

Reply:
resolved